exchanges/ftx/ftx_types.go

```diff
@@ -0,0 +1,966 @@
+package ftx
+
+import (
+	"errors"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+)
+
+// MarketData stores market data
+type MarketData struct {
+	Name           string  `json:"name"`
+	BaseCurrency   string  `json:"baseCurrency"`
+	QuoteCurrency  string  `json:"quoteCurrency"`
+	MarketType     string  `json:"type"`
+	Underlying     string  `json:"underlying"`
+	Change1h       float64 `json:"change1h"`
+	Change24h      float64 `json:"change24h"`
+	ChangeBod      float64 `json:"changeBod"`
+	QuoteVolume24h float64 `json:"quoteVolume24h"`
+	Enabled        bool    `json:"enabled"`
+	Ask            float64 `json:"ask"`
+	Bid            float64 `json:"bid"`
+	Last           float64 `json:"last"`
+	USDVolume24h   float64 `json:"volumeUSD24h"`
+	MinProvideSize float64 `json:"minProvideSize"`
+	PriceIncrement float64 `json:"priceIncrement"`
+	SizeIncrement  float64 `json:"sizeIncrement"`
+	Restricted     bool    `json:"restricted"`
+}
+
+// Markets stores all markets data
+type Markets struct {
+	Result []MarketData `json:"result"`
```

Comment:
Can be optomised even further. Can see how here: https://github.com/thrasher-corp/gocryptotrader/blob/master/exchanges/coinbene/coinbene.go#L78

Reply:
Bump, this can be type Markets []MarketData. Then, you'd have an intermediary struct for capturing `Result` like in Coinbene. Rids redundant code and easier for developers to work with.